nixos/modules/virtualisation/oci-options.nix

```
@@ -9,6 +9,12 @@
           Whether the OCI instance is using EFI.
         '';
       };
+      diskSize = lib.mkOption {
+        type = lib.types.int;
+        default = 8192;
+        description = "Size of the disk image created in MB.";
+        example = "diskSize = 12288; # 12GiB";
```

```suggestion
        example = "diskSize = 12 * 1024; # 12GiB";
```
Less hard for humans. They're bad at division. Monkey patching is easier.

---

Good suggestion. Respun, added you as co-author.